c2corg_ui/templates/route/helpers/view.html

```diff
@@ -170,7 +173,11 @@
        ${get_document_up_down(route, 'height_diff')}
 
         % if route.get('route_length'):
-        <p><span translate class="value-title">route_length</span>: <b class="value">${route['route_length']/1000}&nbsp;km</b></p>
+        <p><span translate class="value-title">route_length</span>: <b class="value">${format_length(route['route_length']) | n}</b></p>
```

Comment:
Should you use ``format_length`` for attributes ``route['mtb_length_asphalt']`` and ``route['mtb_length_trail']`` a few lines below?